backend/workers/queues.js

```diff
@@ -35,21 +36,32 @@ const sendSampleEmailWorker = async job => {
 	});
 };
 
+const deleteAccountWorker = async job => {
+	const { User } = require('../models');
+	const { userID } = job.data;
+	const user = await User.findByPk(userID);
+	return user.destroy();
+};
+
```

Comment:
Consider refactoring these workers and moving each worker to their respective model file. You will need to create a separate email model for this purpose.

Reply:
Will move worker behaviors into their respective models at a later stage.